lib/smart_answer_flows/calculate-your-child-maintenance/questions/gets_benefits.govspeak.erb

```diff
@@ -7,8 +7,27 @@
 <% end %>
 
 <% options(
-  "yes": "Yes",
-  "no": "No"
+  "income_support": "Income Support",
```

Comment:
That is true I opted with going with this option.. it was easier to track the changes from regression test artefacts. 
At this point it may become some what relevant to have them defined in the calculator for this smart answer.